src/cli/onefuzz/api.py

```diff
@@ -274,6 +274,29 @@ def list(self) -> List[responses.ContainerInfoBase]:
         self.logger.debug("list containers")
         return self._req_model_list("GET", responses.ContainerInfoBase)
 
+    def reset(
+        self, container_types: Optional[List[enums.ContainerType]] = None
+    ) -> None:
+        """Reset containers unless specified.
+        [Caution]: It can lead to unexpected results.
```

Comment:
@bmc-msft The cli is not able to read the lines after first one. Some bug exists over there.